homeassistant/components/flipr/config_flow.py

```diff
@@ -34,7 +37,7 @@ async def async_step_user(self, user_input=None):
         self._password = user_input[CONF_PASSWORD]
 
         errors = {}
-        if not self._flipr_id:
+        if self._flipr_id == "":
```

Comment:
I think we can keep it like it was.
```suggestion
        if not self._flipr_id:
```

Reply:
You are right, I corrected it.